app/src/main/kotlin/com/simplemobiletools/gallery/pro/dialogs/PickDirectoryDialog.kt

```diff
@@ -178,6 +180,24 @@ class PickDirectoryDialog(
         }
     }
 
+    private fun getOtherFolderOpeningPath(): String {
+        val lastCopyPath = config.lastCopyPath
+
+        val lastCopyPathExist = {
```

Comment:
do we really need this as a function? Looks quite weird

Reply:
not really. just to make it consistent with `lastCopyPathVisible`. 
now it's updated by removing the function definition.

Reply:
and why is lastCopyPathVisible a function at all too? :) I meant both